knimin/lib/data_access.py

```diff
@@ -2415,6 +2405,58 @@ def set_deposited_ebi(self):
                  WHERE barcode IN %s"""
         self._con.execute(sql, [barcodes])
 
+    def _study_exists(self, study_id):
+        """Cofirms that a study ID exists
+
+        Parameters
+        ----------
+        study_id : int
+
+        Raises
+        ------
+        ValueError
+            If the study ID does not exist
+        """
+        sql = """SELECT EXISTS (SELECT 1 FROM pm.study WHERE study_id = %s)"""
```

Comment:
I found this trick from [here](http://stackoverflow.com/questions/7471625/fastest-check-if-row-exists-in-postgresql). I am not aware whether this actually improves performance. @wasade @josenavas any comments?